### ISSUE TITLE
Steer databases of midi.

### ISSUE THREAD

**Issue by HerrSteinway** [mudcube/MIDI.js#196]
I am a total noob to code, I am only starting out with jquery Php Javascript, phyton and C
And it is quite alot to digest, But your beautifull collaborative work made an old Idea accessable.

So here it is.
I have over 20 hours of midi files named an sorted. 
They have been proven effective, in my own music class I teach.
These files I want to upload them as listed categories to a website, so to have them ready to be drag n dropped on a javascript controlled sheetmusic capable of playing.

Midi on web is accessable thanks to you, 
and Sheet music on web is accessable thanks to another github effort.
Similar feats have been pulled off, by for instance, Ultimate-guitar plus.
But instead charging buckets of money I want this to be Opensource, Access for all!

Making music learning Digestable
Not a Database of youtubedistraction to becoming a Maecenas to the patreon-beggars
neither a glorified Type-course to a Dry-as-hell void without a frame of reference to monitor your own progress.

How do I start this, using your gorgeous midiPlatform?

TL; DR 
Cool midi, brah. How do I combine this with a drag and drop library with a playable sheetmusic function?

**Comment by shivrajsa** [comment]
@HerrSteinway Please check this link from MIDI.js demos : [https://galactic.ink/piano/](https://galactic.ink/piano/), it may help. Could you please share which Sheet Music library you will be using.
